File: leetcode/remove_element.py
def remove_element(nums, val):
    counter = 0
    while counter < len(nums):
        if(nums[counter] is val):
            nums.pop(counter)
        else:
            counter += 1
    return len(nums)


remove_element([3, 2, 2, 3], 3)

File: leetcode/removeElement.js
function removeElement(nums, val) {
  let counter = 0;
  while (counter < nums.length) {
    if (nums[counter] === val) {
      nums.splice(counter, 1);
    } else {
      counter++;
    }
  }
  return nums.length;
}

console.log(removeElement([3, 2, 2, 3], 3));
